model/config.go

```diff
@@ -171,8 +171,10 @@ const (
 	DATA_RETENTION_SETTINGS_DEFAULT_FILE_RETENTION_DAYS     = 365
 	DATA_RETENTION_SETTINGS_DEFAULT_DELETION_JOB_START_TIME = "02:00"
 
-	PLUGIN_SETTINGS_DEFAULT_DIRECTORY        = "./plugins"
-	PLUGIN_SETTINGS_DEFAULT_CLIENT_DIRECTORY = "./client/plugins"
+	PLUGIN_SETTINGS_DEFAULT_DIRECTORY          = "./plugins"
+	PLUGIN_SETTINGS_DEFAULT_CLIENT_DIRECTORY   = "./client/plugins"
+	PLUGIN_SETTINGS_DEFAULT_ENABLE_MARKETPLACE = true
+	PLUGIN_SETTINGS_DEFAULT_MARKETPLACE_URL    = "https://marketplace.integrations.mattermost.com"
```

Comment:
I am guessing this is not yet "live". Just wanted to double check that it will indeed be working by the time this gets merged since it's going to be a default.

Reply:
There's a decent chance this /won't/ be live by the time this get merged (i.e. tomorrow), but we will customize community to point at an instance that is live to allow for initial testing. We're expecting this to be live before we cut a release candidate for v5.16 and update the testing servers.

Reply:
That is correct, the plan is to have it up and running when we test/release 5.16.